test/integration/local/test_vw_serving.py

```diff
@@ -35,16 +35,16 @@ def test_vw_serving(local_instance_type, sagemaker_local_session, docker_image,
                     "MODEL_METADATA_POLLING": "false"}
     
     model = sagemaker.model.Model(
-        image=docker_image,
+        image_uri=docker_image,
         role=role,
         env=environ_vars,
         name="test_env",
         model_data=pretrained_model_vw
         )
     model.deploy(initial_instance_count=1, instance_type=local_instance_type)
 
```

Comment:
The SDK was already updated, according to the VW buildspec: https://github.com/aws/sagemaker-rl-container/blob/master/buildspec-vw.yml#L37
Somehow, this was missed in the last update to the code, because I see that the estimator params are updated to be compatible with sagemaker v2.